tests/benchdnn/doc/knob_strides.md

```diff
@@ -18,14 +18,16 @@ For example, tensor `2x3` can be represented in multiple ways:
 
 ## Usage
 ```
-    --strides=SRC_DIMS[:WEI_DIMS]:DST_DIMS
+    --strides=SRC_DIMS[:OPTIONAL_DIMS]:DST_DIMS
```

Comment:
Please update `benchdnn/utils/parser.cpp:788` place with identical text as well. That's what will show up when `benchdnn --binary --help` is typed.